erpnext/public/js/controllers/taxes_and_totals.js

```diff
@@ -734,7 +737,7 @@ erpnext.taxes_and_totals = class TaxesAndTotals extends erpnext.payments {
 		}
 	}
 
-	update_paid_amount_for_return() {
+	set_total_amount_to_default_mop(){
```

Comment:
In `taxes_and_total.py` there is a function `update_paid_amount_for_return()` which does the same thing. So maybe don't rename the function for consistency.